sys/usb/usbus/cdc/ecm/cdc_ecm_netdev.c

```diff
@@ -59,6 +59,11 @@ static int _send(netdev_t *netdev, const iolist_t *iolist)
     uint8_t *buf = cdcecm->ep_in->ep->buf;
     const iolist_t *iolist_start = iolist;
     size_t len = iolist_size(iolist);
+    /* interface with alternative function ID 1 is the interface containing the
+     * data endpoints, no sense trying to transmit data if it is not active */
+    if (cdcecm->active_iface != 1) {
+        return -EBUSY;
```

Comment:
Is `-EBUSY` the correct error code here? @miri64 Do you have an opinion on this? What should a `_send()` call return when the interface is unable to transmit a PDU?

Reply:
Mhhh... `-EBUSY` was introduced in #11264 to signal the device is busy (even though I abandoned that idea for now for `at86rf2xx` at least). Is that the case here as well? How about `-ENXIO` (No such device or address)?

Reply:
The case here is that either the 'cable' is not connected or the other end of the 'cable' is not configured.

'cable': the point-to-point network data stream over the USB endpoints.

`-EBUSY` is probably wrong in that case, `-ENXIO` is a better match, `-ENOTCONN` could also work and based on a quick look the Linux kernel also seems to use it for the USB gadget framework.

Reply:
replaced `-EBUSY` with `-ENOTCONN`